openfisca_france/parameters/prestations_sociales/prestations_familiales/education_presence_parentale/ars/ars_plaf/plafond_ressources.yaml

```
@@ -133,6 +135,11 @@ metadata:
       href: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000037951022
     - title: Arrêté du 21/12/2022, art. 3
       href: https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000046794210
+    2024-01-01:
+    - title: Article L543-1 du Code de la sécurité sociale
+      href: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000037951022
+    - title: Arrêté du 22/12/2023, art. 1
```

Il me semble que c'est à l'article 3 ?

```suggestion
    - title: Arrêté du 22/12/2023, art. 3
```